consensus/hotstuff/verification/staking_verifier.go

```diff
@@ -92,7 +92,54 @@ func (v *StakingVerifier) VerifyQC(signers flow.IdentityList, sigData []byte, bl
 	}
 
 	if !stakingValid {
-		return fmt.Errorf("invalid aggregated staking sig for block %v: %w", block.BlockID, model.ErrInvalidSignature)
+		return fmt.Errorf("invalid aggregated staking sig for block %v: %w", blockID, model.ErrInvalidSignature)
+	}
+	return nil
+}
+
+// VerifyTC checks cryptographic validity of the TC's `sigData` for the given view.
+// It is the responsibility of the calling code to ensure
+// that all `voters` are authorized, without duplicates. Return values:
+//  - nil if `sigData` is cryptographically valid
+//  - model.ErrInvalidFormat if `sigData` has an incompatible format
+//  - model.ErrInvalidSignature if a signature is invalid
+//  - unexpected errors should be treated as symptoms of bugs or uncovered
+//	  edge cases in the logic (i.e. as fatal)
+func (v *StakingVerifier) VerifyTC(voters flow.IdentityList, sigData []byte, view uint64, highQCViews []uint64) error {
+	return verifyTC(voters, sigData, view, highQCViews, v.stakingHasher)
+}
+
+// verifyTC checks cryptographic validity of the TC's `sigData` for the given view.
+// It is the responsibility of the calling code to ensure
+// that all `voters` are authorized, without duplicates. Return values:
+//  - nil if `sigData` is cryptographically valid
+//  - model.ErrInvalidFormat if `sigData` has an incompatible format
+//  - model.ErrInvalidSignature if a signature is invalid
+//  - unexpected errors should be treated as symptoms of bugs or uncovered
+//	  edge cases in the logic (i.e. as fatal)
+func verifyTC(voters flow.IdentityList, sigData []byte, view uint64, highQCViews []uint64, hasher hash.Hasher) error {
+	if len(voters) == 0 {
+		return model.NewInvalidFormatErrorf("empty list of signers")
```

Comment:
```suggestion
		return model.NewInsufficientSignaturesErrorf("empty list of signers")
```
For consistency with `VerifyQC`. Though alternatively, maybe there is no real reason to differentiate between "invalid format" and "insufficient signers", and we could consolidate on using only `InvalidFormatError` for this case across both TC/QCs.